Makefile

```diff
@@ -7,7 +7,10 @@ endif
 
 all: examples/cpp.out
 
-examples/cpp.out: target/debug/libchallenge_bypass_ristretto.a examples/cpp/main.cpp 
+src/lib.h: src/lib.rs
+	cbindgen -o src/lib.h
```

Comment:
it appears nightly rust is required to regenerate the header, I am able to reproduce locally

Reply:
Can we make a local tweak to enable this to run without a nightly Rust?  I'm nervous about relying on things that are reproducible only on bleeding-edge versions of software rather than stable, widely distributed ones.

Reply:
From the rust 1.30.0 in Ubuntu 18.04, I bootstrapped rust 1.31.0 stable, rust 1.32.0 stable, rust 1.33.0 beta as of 20190211, and rust 1.33.0 nightly as of 20190208, and confirmed that it reproduces the src/lib.h bit-for-bit in the source tree here.  So I'm satisfied (if a little frustrated at the weight and churn of the rust toolchain!).